hack/generate-yamls.sh

```diff
@@ -112,4 +118,6 @@ echo "All manifests generated"
 # List generated YAML files, with serving.yaml first.
 
 ls -1 ${SERVING_YAML} > ${YAML_LIST_FILE}
+ls -1 ${SERVING_CRD_ALPHA_YAML} >> ${YAML_LIST_FILE}
+ls -1 ${SERVING_CRD_BETA_YAML} >> ${YAML_LIST_FILE}
```

Comment:
Just double checking: we'll be releasing the CRDs separately, but it won't be (at least for now) affecting the way we install serving (apply the manifest twice, first for the CRDs, then for the rest).

Reply:
Yes, a docs update will come separately and SERVING_YAML will contain the alpha CRDs because they are symlinked.